### PR TITLE
1332 delete service instances

### PR DESCRIPTION
**Pull request by jimmykarily**
Part of: #1332

### CODE REVIEW

File: internal/api/v1/service/delete.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	err = kubeServiceClient.Delete(ctx, namespace, serviceName)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			return apierror.NewNotFoundError("service not found")
+		}
+
+		return apierror.InternalError(err)
+	}
+
+	response.OKReturn(c, nil)
```
Comment (enrichman) [inline internal/api/v1/service/delete.go:40]: If we are not returning data we can use just `response.OK()`

File: acceptance/services_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+			env.SetupAndTargetNamespace(namespace)
+
+			service = catalog.NewServiceName()
+
+			out, err := env.Epinio("", "service", "create", "mysql-dev", service)
+			Expect(err).ToNot(HaveOccurred(), out)
+
+			out, err = env.Epinio("", "service", "show", service)
+			Expect(err).ToNot(HaveOccurred(), out)
+			Expect(out).To(MatchRegexp("Name.*|.*service-instance-268892044"))
```
Comment (enrichman) [inline acceptance/services_test.go:25]: Does it work with the "fixed" name?

File: acceptance/api/v1/service_delete_test.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+		When("helmchart exists", func() {
+			var serviceName string
+
+			BeforeEach(func() {
+				serviceName = catalog.NewServiceName()
+			})
+
+			When("helmchart is not labeled", func() {
+				BeforeEach(func() {
+					helmChart := helmapiv1.HelmChart{
```
Comment (enrichman) [inline acceptance/api/v1/service_delete_test.go:75]: Should we use the MakeInstance func that you already added with the other PR?

**Review by enrichman — COMMENTED** [review]
Some comments

File: acceptance/api/v1/service_delete_test.go (hunk 3 above)
Comment (jimmykarily) [reply acceptance/api/v1/service_delete_test.go:75]: I need to control the labels in this case (see the `when` description)

Comment (jimmykarily) [reply acceptance/api/v1/service_delete_test.go:75]: I can use the helper in the "labeled" case below though. Changing it.

File: internal/api/v1/service/delete.go (hunk 1 above)
Comment (jimmykarily) [reply internal/api/v1/service/delete.go:40]: true, changing

File: acceptance/services_test.go (hunk 2 above)
Comment (jimmykarily) [reply acceptance/services_test.go:25]: Stupid me. It only worked because the `|` needs to be escaped (otherwise it's a regex "or"). Fixing

**Review by enrichman — APPROVED** [review]

**Pull request merged**